templates/publisher/listing.html

```diff
@@ -27,6 +27,10 @@
           </div>
           <div class="col-5">
             <div class="u-align--right u-clearfix">
+              <button class="js-listing-preview p-button--base u-no-margin--bottom p-tooltip p-tooltip--btm-center" form="preview-form" area-describedby="preview-tooltip">
```

Comment:
TIL about `form` on a `button`.
And I was wondering what kind of magic submits the preview... ;)